src/test/function_test/utils/test_util.h

```diff
@@ -55,7 +55,14 @@ class test_util : public ::testing::Test
 
     void SetUp() override;
 
-    void run_cmd_from_project_root(const std::string &cmd);
+    static void run_cmd_from_project_root(const std::string &cmd);
+
+    // Get the count of alive replica servers.
+    static int get_replica_server_count();
```

Comment:
get_alive_replica_count

Reply:
Renamed to `get_alive_replica_server_count`